.pylintrc

```
@@ -0,0 +1,594 @@
+# 3 compatible code, which means that the block might have code that exists
+# only in one or another interpreter, leading to false positives when analysed.
+analyse-fallback-blocks=no
+
+# Deprecated modules which should not be used, separated by a comma.
+deprecated-modules=optparse,tkinter.tix
+
+# Create a graph of external dependencies in the given file (report RP0402 must
+# not be disabled).
+ext-import-graph=
+
+# Create a graph of every (i.e. internal and external) dependencies in the
+# given file (report RP0402 must not be disabled).
+import-graph=
+
+# Create a graph of internal dependencies in the given file (report RP0402 must
+# not be disabled).
+int-import-graph=
+
+# Force import order to recognize a module as part of the standard
+# compatibility libraries.
+known-standard-library=
+
+# Force import order to recognize a module as part of a third party library.
+known-third-party=enchant
+
+
+[CLASSES]
+
+# List of method names used to declare (i.e. assign) instance attributes.
+defining-attr-methods=__init__,
+                      __new__,
+                      setUp
+
+# List of member names, which should be excluded from the protected access
+# warning.
+exclude-protected=_asdict,
+                  _fields,
+                  _replace,
+                  _source,
+                  _make
+
+# List of valid names for the first argument in a class method.
+valid-classmethod-first-arg=cls
+
+# List of valid names for the first argument in a metaclass class method.
+valid-metaclass-classmethod-first-arg=cls
+
+
+[DESIGN]
+
+# Maximum number of arguments for function / method.
+# BasPH: choose 10 because this was 80% of the sorted list of number of arguments above 5 (Pylint default)
+max-args=10
+
+# Maximum number of attributes for a class (see R0902).
+# BasPH: choose 15 because this was 80% of the sorted list of number of attributes above 7 (Pylint default)
+max-attributes=15
+
+# Maximum number of boolean expressions in an if statement.
+max-bool-expr=5
+
+# Maximum number of branch for function / method body.
+# BasPH: choose 22 because this was 80% of the sorted list of number of attributes above 12 (Pylint default)
+max-branches=22
+
+# Maximum number of locals for function / method body.
+# BasPH: choose 24 because this was 80% of the sorted list of number of locals above 15 (Pylint default)
+max-locals=24
+
+# Maximum number of parents for a class (see R0901).
+max-parents=7
+
+# Maximum number of public methods for a class (see R0904).
+# BasPH: choose 27 because this was 50% of the sorted list of 30 number of public methods above 20 (Pylint default)
+max-public-methods=27
+
+# Maximum number of return / yield for function / method body.
+max-returns=6
+
+# Maximum number of statements in function / method body.
+# BasPH: choose 69 because this was 80% of the sorted list of number of statements above 50 (Pylint default)
+max-statements=50
```

```suggestion
max-statements=69
```
(In case it was a typo)

---

Whoops, fixed